akka-http-core/src/test/scala/akka/testkit/AkkaSpec.scala

```diff
@@ -72,7 +75,7 @@ abstract class AkkaSpec(_system: ActorSystem)
 
   def this() = this(ActorSystem(AkkaSpec.getCallerName(getClass), AkkaSpec.testConf))
 
-  val log: LoggingAdapter = Logging(system, this.getClass)
+  val log: LoggingAdapter = Logging(system, this.getClass.asInstanceOf[Class[Any]])
```

Comment:
I don't really understand this one TBQH...

Reply:
I tried variations but also failed to come up something better. It seems some automatic conversions wrt to Java wildcard types don't work as well (or at least different) in Scala 3 than before.